scripts/crowdin/upload-screenshots.js

```
@@ -0,0 +1,272 @@
+
+    await captureSimulatorPage(page);
+
+    console.log('🎉 Screenshot capture process completed!');
+
+  } finally {
+    await browser.close();
+  }
+}
+
+async function captureSimulatorPage(page) {
+  console.log('🔧 Capturing simulator page with interactions...');
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)